opentreemap/treemap/templates/treemap/partials/udf_edit_field.html

```diff
@@ -9,6 +9,7 @@
 </select>
 {% elif field.type == 'date' %}
 <input class="form-control" type="text" data-field-name="{{ field.name }}"
+       data-date-start-date="{{ datepicker_start_date|date:request.instance.short_date_format|datepicker_format }}"
```

Comment:
As a short term, reasonable fix I would like to try updating the datepicker and enabling `assumeNearbyYear`.